api/RTCPeerConnection.json

```diff
@@ -0,0 +1,3237 @@
+{
+  "api": {
+    "RTCPeerConnection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection",
+        "support": {
+          "webview_android": [
+            {
+              "prefix": "webkit",
+              "version_added": "51",
+              "version_removed": "56"
+            },
+            {
+              "notes": "unprefixed",
```

Comment:
We don't mark unprefixed separately. So this needs to be a block like this, with the unprefixed / "normal" support first:
```json
          "webview_android": [
            {
              "version_added": "56"
            },
            {
              "prefix": "webkit",
              "version_added": "51",
              "version_removed": "56"
            }
          ],
```

This is an issue throughout the file. I'm not marking it everywhere. Please double-check this.